src/games/strategy/triplea/ui/MapPanel.java

```diff
@@ -140,6 +143,16 @@ public void mouseReleased(final MouseEvent e) {
         if (terr != null) {
           notifyTerritorySelected(terr, md);
         }
+        if(e.getButton() == 4 || e.getButton() == 5){
+          if (is4Pressed && is5Pressed) {
```

Comment:
This would be good to extract to a method, simply return the value you need and set it to lastActive directly.